test/gtest/uct/uct_test.cc

```diff
@@ -741,11 +741,11 @@ uct_test::entity::entity(const resource& resource, uct_iface_config_t *iface_con
         if (ifa_addr->sa_family == AF_INET) {
             struct sockaddr_in *addr =
                 reinterpret_cast<struct sockaddr_in *>(ifa_addr);
-            addr->sin_port = ucs::get_port();
+            addr->sin_port = ntohs(ucs::get_port());
```

Comment:
can we use sock_addr_storage::get_port() ?

Reply:
`sock_addr_storage::get_port` does different thing, it just returns already initialized port number. 
we can add static function to `sock_addr_storage` like `get_free_port` if you think it makes sense.

Reply:
@evgeny-leksikov sorry, i've meant set_port

Reply:
yes, will PR to master

Reply:
@yosefe this is `ucs_sock_addr_t uct_test::entity::m_iface_params::mode::sockaddr::listen_sockaddr`, not a `sock_addr_storage`, so seems like it makes sense to keep `iface_params` as a solid field of `uct_test` for now